app/models/game.rb

```diff
@@ -0,0 +1,10 @@
+class Game < ApplicationRecord
+  has_many :participations
+
+  scope :available,
+    # -> { where('(white_player_id = ? AND black_player_id != ?)
+    #             OR
+    #             (white_player_id != ? AND black_player_id = ?)',
+    #             nil, nil, nil, nil) }
+    -> { where('(white_player_id = ? OR black_player_id = ?)', nil, nil) }
```

Comment:
So, two things - first it seems like Postgres is having an issue with the `= nil` syntax and actually wants you to use `IS NULL` and `IS NOT NULL`.  That fixes the first test failure, however the one that omits games with two missing players then fails.  For that you want to find games missing one player or the other but not both.  Something like:

```ruby
class Game < ApplicationRecord
  has_many :participations

  scope :available, -> { 
    where('(white_player_id IS NULL AND black_player_id IS NOT NULL) OR (white_player_id IS NOT NULL AND black_player_id IS NULL)')
  }
end
```